### PR TITLE
Implemented Backend with Express

### LINKED ISSUE #3: Setup Backend directory
Backend setup should be in a node environment with an added web framework support
+ additional config files

### PR DESCRIPTION
**Pull request by sauravbanna**
Implemented Backend with Express

Added a test js file as an example with a GET request to the server in App.tsx

Resolves #3

### CODE REVIEW

File: backend/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+3. make sure you are in the `backend/` directory
+
+4. Run `npm install` to install all relevant packages
+
+## Running the Server
+
+1. make sure you are in the `backend` directory
+
+2. Run `node backend_test.js` to run the server
```
Comment (Michaeltaoma) [inline backend/README.md:23]: small nit: should we use a better name for the backend entry?

File: backend/README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+3. the server will run on `http://localhost:3000/`
+
+###### Note
+
+- _The server needs to be stopped and restarted every time a change is made to the backend JS file
```
Comment (Michaeltaoma) [inline backend/README.md:29]: small nit: extract "_"

File: backend/README.md (hunk 1 above)
Comment (armintalaie) [reply backend/README.md:23]: agreed

File: backend/README.md (hunk 2 above)
Comment (armintalaie) [reply backend/README.md:29]: agreed

**Review by armintalaie — APPROVED** [review]
great work. although something to consider is that our React is with TypeScript. especially with the backend, it can be useful to implement. we can merge this and make a ticket to add TS + autoreload

File: backend/README.md (hunk 1 above)
Comment (sauravbanna) [reply backend/README.md:23]: For sure, this was just a placeholder

**Comment by sauravbanna** [comment]
> great work. although something to consider is that our React is with TypeScript. especially with the backend, it can be useful to implement. we can merge this and make a ticket to add TS + autoreload

I can get started on that, once the issue is made I'll make a new PR

**Comment by Michaeltaoma** [comment]
> 

Hi I have made one [here](https://github.com/ubclaunchpad/tl-dr/issues/18)

**Comment by armintalaie** [comment]
thanks @Michaeltaoma

**Pull request merged**